### ISSUE TITLE
Select collection button missing label on Works page

### ISSUE THREAD

**Issue by rjkati** [samvera/hyrax#6500]
### Descriptive summary

On the Works page, the Select the collection to add your files to box is missing a label

![label_works](https://github.com/samvera/hyrax/assets/32644254/67f86256-12b9-4d1a-b420-4a1120b4464b)

### Steps to reproduce the behavior in User Interface (UI)

1. Install the WAVE accessibility plugin if you have not already done so
2. Go to the Works page: https://pg.nurax.samvera.org/dashboard/my/works?locale=en
3. Click the WAVE plugin to view the report

**Comment by christhepianist** [comment]
Tested and confirmed.

**Comment by rkuehn-uofl** [comment]
Created pull request https://github.com/samvera/hyrax/pull/7070
